packages/web3-providers-ws/src/index.ts

```diff
@@ -28,15 +28,18 @@ import { isNullish, SocketProvider } from 'web3-utils';
 import { ConnectionNotOpenError } from 'web3-errors';
 
 export { ClientRequestArgs } from 'http';
-// todo had to ignore, introduce error in doc generation,see why/better solution
-/** @ignore */
+
 export { ClientOptions } from 'isomorphic-ws';
 
 export default class WebSocketProvider<
 	API extends Web3APISpec = EthExecutionAPI,
 > extends SocketProvider<WebSocket.MessageEvent, WebSocket.CloseEvent, WebSocket.ErrorEvent, API> {
 	protected readonly _providerOptions?: ClientOptions | ClientRequestArgs;
+
 	protected _socketConnection?: WebSocket;
+	public get SocketConnection() {
```

Comment:
changelog update for this is required.

Reply:
Good catch! Thanks,
I just reverted this change and I will do it as part of #5887